src/components/filter/Filter.tsx

```diff
@@ -73,7 +46,7 @@ export const Filter = ({ filterTypes, initialFilters, setQueryVariables }: Props
     filters.initial_start_date && !Object.keys(_omit(filters, Object.keys(initialFilters))).length;
 
   return (
-    <>
+    <Wrapper style={widthSearch && styles.noPadding}>
```

Comment:
if you have conditions for `style` please use an array, as otherwise it could return a boolean value which is not correct for the prop